drivers/openstack/openstack.go

```diff
@@ -736,7 +736,37 @@ func (d *Driver) waitForSSHServer() error {
 		"MachineId": d.MachineId,
 		"IP":        ip,
 	}).Debug("Waiting for the SSH server to be started...")
-	return ssh.WaitForTCP(fmt.Sprintf("%s:%d", ip, d.SSHPort))
+
+	err = ssh.WaitForTCP(fmt.Sprintf("%s:%d", ip, d.SSHPort))
```

Comment:
I like the @ctrlrsf 's solution too. @ehazlett do you wish i replace calls of `WaitForTCP` by the new `WaitForSSH ` in all drivers in this PR ?